data/xml/2020.eamt.xml

```diff
@@ -0,0 +1,728 @@
+      <url hash="79c3e33d">2020.eamt-1.28</url>
+    </paper>
+    <paper id="29">
+      <title>Terminology-Constrained Neural Machine Translation at <fixed-case>SAP</fixed-case></title>
+      <author><first>Miriam</first><last>Exel</last></author>
+      <author><first>Bianka</first><last>Buschbeck</last></author>
+      <author><first>Lauritz</first><last>Brandt</last></author>
+      <author><first>Simona</first><last>Doneva</last></author>
+      <pages>271–280</pages>
+      <abstract>This paper examines approaches to bias a neural machine translation model to adhere to terminology constraints in an industrial setup. In particular, we investigate variations of the approach by Dinu et al. (2019), which uses inline annotation of the target terms in the source segment plus source factor embeddings during training and inference, and compare them to constrained decoding. We describe the challenges with respect to terminology in our usage scenario at SAP and show how far the investigated methods can help to overcome them. We extend the original study to a new language pair and provide an in-depth evaluation including an error classification and a human evaluation.</abstract>
+      <url hash="199ff6ff">2020.eamt-1.29</url>
+    </paper>
+    <paper id="30">
+      <title>Ellipsis Translation for a Medical Speech to Speech Translation System</title>
+      <author><first>Jonathan</first><last>Mutal</last></author>
+      <author><first>Johanna</first><last>Gerlach</last></author>
+      <author><first>Pierrette</first><last>Bouillon</last></author>
+      <author><first>Hervé</first><last>Spechbach</last></author>
+      <pages>281–290</pages>
+      <abstract>In diagnostic interviews, elliptical utterances allow doctors to question patients in a more efficient and economical way. However, literal translation of such incomplete utterances is rarely possible without affecting communication. Previous studies have focused on automatic ellipsis detection and resolution, but only few specifically address the problem of automatic translation of ellipsis. In this work, we evaluate four different approaches to translate ellipsis in medical dialogues in the context of the speech to speech translation system BabelDr. We also investigate the impact of training data, using an under-sampling method and data with elliptical utterances in context. Results show that the best model is able to translate 88
+
+% of elliptical utterances.</abstract>
```

Comment:
blank line

Reply:
This was `...able to translate 88\\% of elliptical utterances.",` in the BibTeX, FYI. I'll just correct manually.